### DISCUSSION TITLE
问题咨询：添加实例执行delete工单提示Execute Successfully Backup failed

### DISCUSSION THREAD

**Discussion by dbatool** [hhyo/Archery#1642] [Q&A] [answered]
### 问题是否已经被报告?

- [x] 我已经搜索了当前存在的问题

### 重现步骤

1.添加实例，配置的SQL账号权限为：GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, ALTER, REPLICATION CLIENT ON * . * TO `user_archery`@`x.x.x.x`;
2.提交一个 SQL上线的工单，内容：delete from test.tb_sys_log where id='998'
3.正常审批，在最后提示error，详细错误如下：
审核/执行状态：error
审核/执行信息：Backup: ERROR 1045 (28000): Access denied for user 'user_archery'@'x.x.x.x' (using password: YES).
当前阶段：Execute Successfully Backup failed

### 预期外的结果

请问提交工单所在的SQL实例，这个账号权限应该赋予什么权限？我已经配置很高的权限了，谢谢各位。

### 日志文本

```shell
Backup: ERROR 1045 (28000): Access denied for user 'user_archery'@'x.x.x.x' (using password: YES).
```

### 版本

1.8.5

### 部署方式

Docker

### 是否还有其他可以辅助定位问题的信息？比如数据库、浏览器、goInception等

请问提交工单所在的SQL版本为：MySQL 8.0.23

**Comment by nick2wang** [comment, marked as answer]
大概率是goinception配置文件中的备份库信息不对，除了配置项管理中的备份库信息，还需要手动修改config.toml
https://github.com/hhyo/archery/wiki/configuration#goInception%E9%85%8D%E7%BD%AE

**Comment by MapFishCui** [comment]
缺少REPLICATION SLAVE权限
